### ISSUE TITLE
Parser Errors: Unexpected '/'.

### ISSUE THREAD

**Issue by kaysonwu** [kristerkari/stylelint-scss#471]
```less
.select {
  // For undefined value baseline align
  &::after,
  // For '' value baseline align
  .@{select-prefix-cls}-selection-item::after,
  // For common baseline align
  .@{select-prefix-cls}-selection-placeholder::after {
    display: inline-block;
    width: 0;
    visibility: hidden;
    content: '\a0';
  }
}
```

Are comments not allowed in selectors？

![error](https://user-images.githubusercontent.com/14865584/82283508-249d8280-99c9-11ea-8d0c-991385c8041c.png)

**Comment by kristerkari** [comment]
Thanks, I'll have a look at it.

**Comment by kristerkari** [comment]
@kaysonwu I had a look at this, and it looks like your code is getting parsed as something else than Less (either CSS or SCSS), and that's why it's throwing an error. 

Can you check if your linting settings actually specify that you are using Less?
